pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -110,7 +111,7 @@ def work(self, response_dict=None):
         )
 
         # simulate app
-        sleep(3)
+        action_delay(self.config.catchsim_catch_wait_min, self.config.catchsim_catch_wait_max)
```

Comment:
This should be app simulation not waiting to throw a ball.  The app will be quite consistent in the amount of time it takes to play an animation.  sleep(3) should be fine but if you want to expose the option it needs to be a separate option.